src/github.sh

```diff
@@ -13,7 +13,15 @@ github::get_pr_number() {
 }
 
 github::get_pr_body() {
-  jq --raw-output .pull_request.body "$GITHUB_EVENT_PATH"
+  jq --raw-output .pull_request.body "$GITHUB_EVENT_PATH" | sed -r 's/#{1,3}/####/'
```

Comment:
someone with more than cursory knowledge of sed might wanna take a look at this 😅